docs/crio.8.md

```diff
@@ -274,28 +272,30 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--root, -r**="": The CRI-O root directory (default: /var/lib/containers/storage)
 
-**--runroot**="": The CRI-O state directory (default: /run/containers/storage)
+**--runroot**="": The CRI-O state directory (default: /var/run/containers/storage)
```

Comment:
The changes in the file that aren't related to `stream-idle-timeout` should be dropped.